validator/keymanager/v2/types.go

```diff
@@ -0,0 +1,47 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/prysmaticlabs/prysm/shared/bls"
+)
+
+// IKeymanager defines a general keymanager-v2 interface for Prysm wallets.
+type IKeymanager interface {
+	// CreateAccount based on the keymanager's logic.
+	CreateAccount(ctx context.Context, password string) error
+	// MarshalConfigFile for the keymanager's options.
+	MarshalConfigFile(ctx context.Context) ([]byte, error)
+	// FetchValidatingKeys fetches the list of public keys that should be used to validate with.
+	FetchValidatingPublicKeys() ([][48]byte, error)
```

Comment:
Why is this `FetchValidatingPublicKeys` vs. `FetchPublicKeys`? Can the wallet distinguish the two?

Reply:
We want to make sure it is clear these are the validating public keys and not withdrawal public keys